ui/00_message/achievement/achievement_info.xml

```diff
@@ -6645,50 +6653,62 @@ level 30 in crafting</translation>
   <message>
     <key>ACHIEVEMENT_INFO_2537</key>
     <original>エラン水林で&lt;ITEM 16522&gt;を15個発見</original>
+    <translation>Find 15 &lt;ITEM 16522&gt; in Elan Water Grove</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2538</key>
     <original>エラン水林で&lt;ITEM 16522&gt;を30個発見</original>
+    <translation>Find 30 &lt;ITEM 16522&gt; in Elan Water Grove</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2539</key>
     <original>ファーラナ平原で&lt;ITEM 16522&gt;を15個発見</original>
+    <translation>Find 15 &lt;ITEM 16522&gt; in Farana Plains</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2540</key>
     <original>ファーラナ平原で&lt;ITEM 16522&gt;を30個発見</original>
+    <translation>Find 30 &lt;ITEM 16522&gt; in Farana Plains</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2541</key>
     <original>モロー樹林で&lt;ITEM 16522&gt;を15個発見</original>
+    <translation>Find 15 &lt;ITEM 16522&gt; in Morrow Forest</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2542</key>
     <original>モロー樹林で&lt;ITEM 16522&gt;を30個発見</original>
+    <translation>Find 30 &lt;ITEM 16522&gt; in Morrow Forest</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2543</key>
     <original>キンガル峡谷で&lt;ITEM 16522&gt;を15個発見</original>
+    <translation>Find 15 &lt;ITEM 16522&gt; in Kingal Canyon</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2544</key>
     <original>キンガル峡谷で&lt;ITEM 16522&gt;を30個発見</original>
+    <translation>Find 30 &lt;ITEM 16522&gt; in Kingal Canyon</translation>
   </message>
   <message>
     <key>ACHIEVEMENT_INFO_2545</key>
     <original>&lt;ITEM 16522&gt;、&lt;ITEM 18738&gt;を累計30個発見</original>
+    <translation>Found a total of 30&lt;ITEM 16522&gt;、&lt;ITEM 18738&gt;</translation>
```

Comment:
Don't use the unicode "、" in translations, change it to a regular ",". And be careful of spacing, it will not add a space before the item name here.